Invalid use of builtin triggers a panic

The following program `sample.go` triggers a panic:

```go
package main

func main() {
    f := println
    f("hello")
}
```

Expected result:
```console
$ go run ./sample.go
_test/bltn.go:4:7: use of builtin println not in function call
```

Got:
```console
$ dyngo ./sample.go
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0x11e718a]

goroutine 1 [running]:
github.com/containous/dyngo/interp.(*Type).TypeOf(0x0, 0x1, 0x1)
	/Users/marc/go/src/github.com/containous/dyngo/interp/type.go:458 +0x3a
github.com/containous/dyngo/interp.isInt(0x0, 0x213d000)
	/Users/marc/go/src/github.com/containous/dyngo/interp/type.go:510 +0x2b
github.com/containous/dyngo/interp.(*Interpreter).Cfg.func2(0xc0001dad80)
	/Users/marc/go/src/github.com/containous/dyngo/interp/cfg.go:277 +0x4b65
github.com/containous/dyngo/interp.(*Node).Walk(0xc0001dad80, 0xc0001fbd80, 0xc0001fbd40)
	/Users/marc/go/src/github.com/containous/dyngo/interp/interp.go:114 +0xa5
github.com/containous/dyngo/interp.(*Node).Walk(0xc0001dac60, 0xc0001fbd80, 0xc0001fbd40)
	/Users/marc/go/src/github.com/containous/dyngo/interp/interp.go:111 +0x66
github.com/containous/dyngo/interp.(*Node).Walk(0xc0001da6c0, 0xc0001fbd80, 0xc0001fbd40)
	/Users/marc/go/src/github.com/containous/dyngo/interp/interp.go:111 +0x66
github.com/containous/dyngo/interp.(*Node).Walk(0xc0001da480, 0xc0001fbd80, 0xc0001fbd40)
	/Users/marc/go/src/github.com/containous/dyngo/interp/interp.go:111 +0x66
github.com/containous/dyngo/interp.(*Interpreter).Cfg(0xc0001e4070, 0xc0001da480, 0x0, 0x0, 0x0, 0x0, 0x4)
	/Users/marc/go/src/github.com/containous/dyngo/interp/cfg.go:141 +0x1af
github.com/containous/dyngo/interp.(*Interpreter).Eval(0xc0001e4070, 0xc0001e8000, 0x38, 0x188fa4f, 0x4, 0xc0001e4070, 0x0, 0x178ada0)
	/Users/marc/go/src/github.com/containous/dyngo/interp/interp.go:208 +0xfb
main.main()
	/Users/marc/go/src/github.com/containous/dyngo/dyngo.go:50 +0x334
```

--- a/interp/cfg.go
+++ b/interp/cfg.go
@@ -650,7 +650,6 @@ func (interp *Interpreter) Cfg(root *Node) ([]*Node, error) {
 
 		case GoStmt:
 			wireChild(n)
-			// TODO: should error if call expression refers to a builtin
 
 		case Ident:
 			if isKey(n) {
@@ -696,6 +695,10 @@ func (interp *Interpreter) Cfg(root *Node) ([]*Node, error) {
 							n.fsize = n.typ.rtype.NumOut()
 						}
 						n.rval = sym.val.(reflect.Value)
+					case sym.kind == Bltn:
+						if n.anc.kind != CallExpr {
+							err = n.cfgError("use of builtin %s not in function call", n.ident)
+						}
 					}
 				}
 				if n.sym != nil {

--- a/interp/interp_consistent_test.go
+++ b/interp/interp_consistent_test.go
@@ -33,7 +33,6 @@ func TestInterpConsistency(t *testing.T) {
 			file.Name() == "cli1.go" || // FIXME global vars
 			file.Name() == "interface0.go" || // TODO not implemented yet
 			file.Name() == "heap.go" || // TODO not implemented yet
-			file.Name() == "bltn.go" || // TODO not implemented yet
 			file.Name() == "chan6.go" || // FIXME related to channel #7
 			file.Name() == "select1.go" || // FIXME related to channel #7
 			file.Name() == "ret1.go" || // TODO not implemented yet #22

